app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/catalog/ui/CatalogScreen.kt

```diff
@@ -24,6 +25,9 @@ import com.microsoft.device.samples.dualscreenexperience.presentation.catalog.ut
 import com.microsoft.device.samples.dualscreenexperience.presentation.catalog.utils.rememberViewPagerState
 import kotlinx.coroutines.launch
 import kotlin.math.abs
+import kotlin.math.absoluteValue
+
+private const val DEFAULT_FRACTION = 1
```

Comment:
If this is just used inside the `calculateLinearInterpolation` function, then it is better to have it just there.